app/gimlet/rev-b.toml

```diff
@@ -100,6 +100,14 @@ notifications = ["i2c1-irq", "jefe-state-change"]
 "i2c1.event" = "i2c1-irq"
 "i2c1.error" = "i2c1-irq"
 
+[tasks.packrat]
+name = "task-packrat"
+priority = 3
+max-sizes = {flash = 8192, ram = 2048}
+start = true
+# task-slots is explicitly empty: packrat should send not send IPCs!
```

Comment:
```suggestion
# task-slots is explicitly empty: packrat should not send IPCs!
```
(also needs fixing in other `app.toml` files)

Reply:
thanks, fixed in 72f64225